api/handler/copy.go

```diff
@@ -109,15 +109,17 @@ func (h *handler) CopyObjectHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	settings, err := h.obj.GetBucketSettings(r.Context(), dstBktInfo)
-	if err != nil {
+	if err != nil && !errors.IsS3Error(err, errors.ErrNoSuchKey) {
 		h.logAndSendError(w, "could not get bucket settings", reqInfo, err)
 		return
 	}
```

Comment:
I think we can refactor `h.obj.GetBucketSettings` so it handle `errors.ErrNosuchKey`

Reply:
Do you mean that `h.obj.GetBucketSettings` should return `nil, nil` if no settings file is found?

Reply:
No, it should return `&data.BucketSettings{}, nil` if settings file isn't found